plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -173,6 +173,13 @@ func defaultPredicates() sets.String {
 				return predicates.NewVolumeNodePredicate(args.PVInfo, args.PVCInfo, nil)
 			},
 		),
+		// Fit is determined by PVC not being deleted
+		factory.RegisterFitPredicateFactory(
```

Comment:
The way I added an alpha predicate in my volume scheduling PR was to only register it if the feature gate is set.

Reply:
Nevermind, that method doesn't seem to work.  Ignore me.